components/layout/PageHeader.tsx

```diff
@@ -10,7 +10,7 @@ import { ErrorBoundary } from '../errors/ErrorBoundary';
 
 function PageHeader() {
   return (
-    <Flex direction={'row'} bg={theme.light.header.background.color} pl="120px" pr="120px">
+    <Flex direction={'row'} bg={theme.light.header.background.color} pr={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }}>
```

Comment:
we should be able to use `rem` or `em` or `vh`/`vw`. Is there a reason we didn't?